docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -188,34 +200,29 @@ The `request_ip` field is optional.
 
 ## Step 4: Embed Checkout
 
-Use the `embedded_checkout_url` that is returned and assemble a JSON object that will be used by the Checkout JS SDK to determine how to render the checkout. Pass the object to the `embedCheckout` method of the Checkout SDK. This will render the checkout to an HTML element with the `id` you chose.
+Use the `embedded_checkout_url` that is returned from generating redirect URLs and assemble a JSON object. It will be used by the Checkout JS SDK to determine how to render the checkout. Pass the object to the `embedCheckout` method of the Checkout SDK. This will render the checkout to an HTML element with the `id` you chose.
 
 Read more about the [JSON object](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/README.md#embedcheckout) and its possible corresponding [rendering options](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutoptions.md).
 
-```html
-<div id="foo-bar-checkout"></div>
-```
+**embedCheckout method**
 
```

Comment:
This needs a sentence to explain what we want the reader to get from the following information.